src/core/io/src/4C_io_discretization_visualization_writer_mesh.hpp

```diff
@@ -81,6 +84,9 @@ namespace Core::IO
         const Core::LinAlg::MultiVector<double>& result_data, const OutputEntity output_entity,
         const std::vector<std::optional<std::string>>& context);
 
+    void append_field_data_vector(
```

Comment:
Please add documentation to this this function (have a look a how the other functions in this file).